WordPressKit/WordPressOrgXMLRPCApi.swift

```diff
@@ -451,3 +506,109 @@ extension WordPressOrgXMLRPCApiError: LocalizedError {
         }
     }
 }
+
+public struct WordPressOrgXMLRPCApiFault: LocalizedError, HTTPURLResponseProviding {
+    var response: HTTPAPIResponse<Data>
+
+    var code: Int?
+    var message: String?
```

Comment:
Do these need to be mutable?

Reply:
Addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/719/commits/c4f376df1ede3867179c59c7b0a9fc4da7a79358